students/sbolsen/lesson03/slice.py

```diff
@@ -0,0 +1,33 @@
+def first_last_exchanged(sequence):
+    sequence[0], sequence[-1] = sequence[-1], sequence[0]
+    return sequence
+
+def elements_reversed(sequence):
+    return sequence[::-1]
+
+def thirds(sequence):
+    a_list = []
+    for i in sequence:
+        a_list.append(i)
+    section = int(len(a_list) / 3)
+    first = a_list[:section]
+    third = a_list[section * 2:section * 3]
+    second = a_list[section:section * 2]
+    return second + third + first
+
+def every_other(sequence):
+    return sequence[1::2]
```

Comment:
this is technically: `sequence[::2]` (first one should be zero but we omit because it's the default)